ecl/resource_ecl_provider_connectivity_tenant_connection_request_v2.go

```diff
@@ -0,0 +1,274 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/ecl/provider_connectivity/v2/tenant_connection_requests"
+	"github.com/nttcom/eclcloud/ecl/sss/v1/approval_requests"
+)
+
+func resourceProviderConnectivityTenantConnectionRequestV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceProviderConnectivityTenantConnectionRequestV2Create,
+		Read:   resourceProviderConnectivityTenantConnectionRequestV2Read,
+		Update: resourceProviderConnectivityTenantConnectionRequestV2Update,
+		Delete: resourceProviderConnectivityTenantConnectionRequestV2Delete,
+		Schema: map[string]*schema.Schema{
+			"tenant_id_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
If an user updates it, resource needs to be recreated. So you need to set ForceNew.
```suggestion
				Required: true,
				ForceNew: true,
```

Reply:
understood.But tenant_id_other does not update.
Is it a parameter of the update method that attaches ForceNew?